roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -108,7 +108,9 @@ class RooAbsReal : public RooAbsArg {
 
   virtual Double_t getValV(const RooArgSet* normalisationSet = nullptr) const ;
 
-  virtual RooSpan<const double> getValBatch(std::size_t begin, std::size_t maxSize, const RooArgSet* normSet = nullptr) const;
+  // getValBatch() is slow and depricated. Please use getValues() instead. If your code is affected by this change,
+  // please consult the release notes for ROOT 6.24 for a quick and easy walkthrough on how to make this transition.
```

Comment:
Please use three slashes, so doxygen can display it on the [website](https://root.cern.ch/doc/master/classRooAbsReal.html).
```
/// \deprecated getValBatch() has been removed in favour of getValues(). If your code is affected by this change,
/// please consult the release notes for ROOT 6.24 for guidance on how to make this transition.
```
I prefer to not say that something is quick and easy. Some people might disagree. :-)